### ISSUE TITLE
feat: support layers for all chart types

### ISSUE THREAD

**Issue by ellvix** [xability/maidr-legacy#523]
Like scatter and line, but for any plot types and any number. This works in theory, but it's never been done or tested so it needs debugging.

**Comment by ellvix** [comment]
@jooyoungseo we need plots with multiple layers to test this properly. Can you create or assign creation of multi layer charts for all types except line / scatter?

**Comment by jooyoungseo** [comment]
@ellvix -- Since there are a bunch of combinations are available and one layer is independent of another, we cannot create all the testable combinations. We can at least ask ChatGPT to create some lulti-layered ggplot and export it as an SVG via svglite package.

**Comment by ellvix** [comment]
@jooyoungseo yes of course, just need at least 1 of each chart represented, not all combos.
